compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -0,0 +1,77 @@
+package nodeinventorizer
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/scanner/database"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+// NodeInventorizer is the interface that defines the interface a scanner must implement
+type NodeInventorizer interface {
+	Scan(nodeName string) (*storage.NodeInventory, error)
+}
+
+// NodeInventoryCollector is the implementation of NodeInventorizer
+type NodeInventoryCollector struct {
+}
+
+// Scan scans the current node and returns the results as storage.NodeInventory object
+func (n *NodeInventoryCollector) Scan(nodeName string) (*storage.NodeInventory, error) {
+	componentsHost, err := nodes.Analyze(nodeName, "/host/", false)
```

Comment:
nit: `hostComponents`?

Reply:
Also maybe comment why we use `false` here

Reply:
Also, since we have a log to say when we finish, can we have a log to state when we start?

Reply:
I've updated the code, added a more detailed comment and update the loglevel